src/TableOfContents.jl

```diff
@@ -197,6 +197,7 @@ const toc_css = """
 
 .plutoui-toc.aside.hide {
 	transform: translateX(calc(100% - 28px));
+	height: 57px;
```

Comment:
Love this! Instead of hide, do we have some HTML structure we can `display:none` on?

Reply:
You can keep the height too (and maybe add animation! (css transform)), on top of the `display:none`